.lokalise.yml

```diff
@@ -0,0 +1,2 @@
+project-id: 331553825e9baabce4a297.57936175
+token: 2c6ba3a71fcf8cb305971cb7822bf8c510794d26 # read-only token
```

Comment:
there's an id and token in this file here. what is this token for?
is it okay to leave it like this in an opensource project?

Reply:
It's read-only so it doesn't uncover anything that isn't already in the exported files.

Alternatively we could use an environment variable and you would need to create your own read only token on lokalise:

`LOKALISE_TOKEN=<my-token> yarn i18n:pull`

Actually, lets go with that.